app/views/search/new.html.slim

```diff
@@ -1,21 +1,29 @@
-h1.title
-  | Search#new
+.row
+  .col-sm-12.col-md-10.col-md-1.col-lg-8.col-lg-offset-2.text-center
+    = form_for new_search_path, method: :get, html: { class: 'form-horizontal' } do |form|
+      .form-group
+        .col-xs-8
+          = text_field_tag :query, params[:query], placeholder: 'Enter Search Term', class: 'form-control input-lg'
+        .col-xs-4
+          = submit_tag 'Search', name: nil, class: 'col-lg-12 btn btn-lg btn-primary'
 
-.customer-search-form
-  = form_for new_search_path, method: :get do |form|
-    = form.label 'Search for'
-    p
-    = text_field_tag :query, params[:query]
-    = submit_tag 'Go', name: nil
 
-ul.search-results
+- unless @results.empty?
```

Comment:
Not now, but we should return a `NullResult` object at some point, which is just an empty array. 

https://robots.thoughtbot.com/rails-refactoring-example-introduce-null-object